projects/schematics/src/add-spartacus/schema.ts

```diff
@@ -5,4 +5,5 @@ export interface Schema {
   useMetaTags?: boolean;
   featureLevel?: string;
   overwriteAppComponent?: boolean;
+  autoBaseSite?: boolean;
```

Comment:
For 2.0 we aim the feature of auto base site to be enabled by default, when you don't pass `--baseSite`.
Can we create a ticket labeled for 2.0 and paste a link it in a code comment here for reference?